.github/vale/styles/Klaw/branding.yml

```diff
@@ -24,4 +24,5 @@ swap:
   "type[ -]?scripts?": TypeScript
   "(?i)ssl": SSL
   "(?<!Klaw )cluster api": Klaw Cluster API
-  "(?i)kafka[ -]?connect": Kafka Connect
\ No newline at end of file
+  "(?i)kafka[ -]?connect": Kafka Connect
+  "(?i)api": API
```

Comment:
Could you remove that line from the `branding-warning-temp` file, too? We don't need it there anymore, since it's now a proper error.